spec/system/provider_interface/candidate_receives_email_when_an_application_has_been_rejected_by_default_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.feature 'An application has been rejected by default' do
```

Comment:
Perhaps we can rewrite https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/system/provider_interface/provider_receives_email_when_an_application_has_been_rejected_by_default_spec.rb to make it test for both emails? Perhaps a single test `app/system/reject_by_default_spec.rb` for the whole feature.

We can even move https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/system/provider_interface/provider_receives_email_when_an_application_is_getting_close_to_the_reject_by_default_date_spec.rb into that.

Reply:
yes that's much better, 3-into-1.